turn-io-adapter/src/main/java/org/egov/rb/service/TransformService.java

```diff
@@ -0,0 +1,70 @@
+package org.egov.rb.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.egov.common.contract.request.RequestInfo;
+import org.egov.rb.contract.MessageRequest;
+import org.egov.rb.pgrmodels.Address;
+import org.egov.rb.pgrmodels.AddressDetail;
+import org.egov.rb.pgrmodels.Citizen;
+import org.egov.rb.pgrmodels.Service;
+import org.egov.rb.pgrmodels.Service.SourceEnum;
+import org.egov.rb.pgrmodels.ServiceRequest;
+import org.egov.rb.pgrmodels.ServiceResponse;
+import org.egov.rb.repository.ServiceRequestRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+
+import lombok.extern.slf4j.Slf4j;
+
+@org.springframework.stereotype.Service
+@Slf4j
+public class TransformService {
+
+	@Value("${egov.pgr.host}")
+	private String pgrBasePath;
+
+	@Value("${egov.pgr.create.endpoint}")
+	private String pgrCreateEndPoint;
+
+	@Autowired
+	ServiceRequestRepository serviceRequestRepository;
+
+	
+	public ServiceResponse transform(MessageRequest messageRequest) {
+		ServiceRequest servicerequest = prepareServiceRequest(messageRequest);
+		StringBuilder url = new StringBuilder(pgrBasePath + pgrCreateEndPoint);
+		ServiceResponse serviceResponse = serviceRequestRepository.fetchResult(url, servicerequest);
+		return serviceResponse;
+
+	}
+
+	private ServiceRequest prepareServiceRequest(MessageRequest messageRequest) {
```

Comment:
Please add comments on top of all functions

Reply:
added comments